### PR TITLE
feat(install): script and docs update

### PR DESCRIPTION
**Pull request by pilartomas**
Version bump will now require modification of the install script. We can eventualy create some auto-patch solution for it.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -17,11 +17,18 @@ then
         exit 1
     fi
 else
-    echo "Node detected"
+    DETECTED="$(node -v)"
+    echo $DETECTED
```
Comment (Tomas2D) [inline install.sh:21]:
```suggestion
echo "Detected version: $DETECTED"
```

File: install.sh [hunk 2]
```diff
@@ -17,11 +17,18 @@ then
...
 else
-    echo "Node detected"
+    DETECTED="$(node -v)"
+    echo $DETECTED 
+    MINIMAL="v16.10.0"
+    SORTED=($(for VER in "$DETECTED" "$MINIMAL"; do echo "$VER"; done | sort -V))
+    if [ "${SORTED[0]}" = "$DETECTED" ]; then
+        echo "Minimal supported version is" $MINIMAL
+        exit 1
+    fi
```
Comment (Tomas2D) [inline install.sh:27]: If we already now that the user has NodeJS installed I would do this.

```shell
MINIMAL="v16.10.0"
DETECTED=`node -v`

RESULT=`node -e "console.log(\"$DETECTED\".replace('v', '').localeCompare(\"$MINIMAL\".replace('v', ''), undefined, { numeric: true }))"`

if [ $RESULT -lt 0 ]; then
        echo "Minimal supported version is" $MINIMAL
        exit 1
fi

**Review by Tomas2D — CHANGES_REQUESTED** [review]

File: install.sh (hunk 1 above)
Comment (pilartomas) [reply install.sh:21]: Not suitable, the version finished an existing line ` blabla ... v1.1.1`

Comment (Tomas2D) [reply install.sh:21]: I see.

File: install.sh (hunk 2 above)
Comment (pilartomas) [reply install.sh:27]: 👍 but I'll rather avoid `localCompare`

**Review by Tomas2D — APPROVED** [review]

**Pull request merged**